src/Omnipay/Common/Helper.php

```diff
@@ -33,6 +34,26 @@ function ($match) {
     }
 
     /**
+     * Convert strings with underscores to be all lowercase before camelCase is preformed.
+     *
+     * @param  string $str The input string
+     * @return string The output string
+     */
+    protected static function convertToLowercase($str)
```

Comment:
The issue had to do with a string without underscores. If `strtolower($str)` was the only thing called it breaks compatibly for already camel cased strings that get passed to `camelCase()`.